src/SprykerEco/Zed/AdyenApi/Business/AdyenApiFacadeInterface.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AdyenApi\Business;
+
+use Generated\Shared\Transfer\AdyenApiRequestTransfer;
+use Generated\Shared\Transfer\AdyenApiResponseTransfer;
+
+interface AdyenApiFacadeInterface
+{
+    /**
+     * Specification:
+     * - Queries the available payment methods for a transaction based on the transaction context.
+     * - Giving back a list of the available payment methods.
```

Comment:
maybe use consistent tenses in the spec. Now you have (1) give*s* (2) Return*ing* (3) read*s*